All project code/views/pages/search.ejs

```diff
@@ -0,0 +1,22 @@
+<%- include ('../partials/header') %>
+    <%- include ('../partials/menu') %>
+
+        <main>
+            <div class="container">
+                <!--  -->
+                <h1>Visit Page</h1>
+                <form action="/search" method="POST">
+
+                    <% searchFields.forEach(searchField => { %>
+                        <label for="<%= searchField.name %>" class="form-label"><%= searchField.label %></label>
+                        <input type="text" class="form-control" id="<%= searchField.name %>" name="<%= searchField.name %>" 
```

Comment:
Don't see the point to having a frontend implementation that will never be used by users

Reply:
Discussed in communication elsewhere:
For now, I'll put this in a different file and get request than /search, for one specific to weather. This can be removed once we have a functional alternative that is more user friendly. For now, though, this is the best (only) access users have to the API, so I think it's worth keeping in.